draft-ietf-httpbis-incremental.md

```diff
@@ -110,7 +110,9 @@ Upon receiving a header section that includes an Incremental header field with a
 true value, HTTP intermediaries SHOULD NOT buffer the entire message before
 forwarding it.  Instead, intermediaries SHOULD transmit the header section
 downstream and continuously forward the bytes of the message body as they
-arrive.
+arrive. As the Incremental header field indicates only how the message body is
```

Comment:
```suggestion
arrive. As the Incremental header field indicates only how the message content is
```

As far as I know, _content_ is to be preferred over _body_: https://httpwg.org/admin/editors/style-guide#content

Reply:
Nice catch, thank you!